### ISSUE TITLE
Timeout for Kudos

### ISSUE THREAD

**Issue by jamesqquick** [Learn-Build-Teach/learn-build-teach-discord-bot#48]
What protection do we need in place to prevent spamming of kudos?

**Comment by mhuggins7278** [comment]
We could update the Kudo model to include the messageId for the reaction kudo's and only allow 1 per user per message not sure the slash commands are as easy to abuse.

**Comment by jamesqquick** [comment]
Yeah we could do that. Are there any other ones that you think would be helpful. People were asking about limiting the amount of kudos a user could give in a day as an example. I'm thinking we just hold off on that until we see it as a problem.

**Comment by mhuggins7278** [comment]
Yeah I was kinda thinking the same. I think we could address is it becomes
problematic but let's not over optimize.
